src/app/python/core.py

```diff
@@ -581,41 +582,28 @@ def _inner_task_apply_changes(self, path):
 
     @param path: The path of the file on which modifications will be applied
     """
-    try:
-      if path in self._project_files:
-        version, changes = self._project_files[path].file.writeModifications()
-        users_registered = deepcopy(self._project_files[path].users)
-
-        # Notify registered users
-        self._notify_event(
-          lambda l: l.notify_file_edit(path,
-                                       changes,
-                                       version,
-                                       users_registered))
-    except:
-      e = sys.exc_info()
-      # XXX Remove after correction! C++ Should handle this!
-      self._logger.exception("EXCEPTION RAISED {0}\n{1}\n{2}".format(e[0], e[1], e[2]))
+    if path in self._project_files:
+      version, changes = self._project_files[path].file.writeModifications()
```

Comment:
Since the try catch is removed, the thread will survive uppon exception.
However, the ZT will remain in error since the erroneous change will still be there